### PR TITLE
DOC added Binding Modes topic to guide and to TOC with associated PNGs

### PR DESCRIPTION
**Pull request by bryanzakounity**
### Purpose of this PR

CMCL-198 To improve binding mode documentation.

Added screenshots of yaw, pitch and roll for each of the binding modes.

### CODE REVIEW

File: Documentation~/CinemachineBindingModes.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# Binding Modes
+
+The binding mode is the coordinate space Unity uses to interpret the Camera offset from the target and the damping.
```
Comment (glabute) [inline Documentation~/CinemachineBindingModes.md:3]: Not _is_.  The binding mode _defines_ the coordinate space

File: Documentation~/CinemachineBindingModes.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+</br></br>
+
+### Lock To Target
+
+Makes the virtual Camera use the local frame of the Follow target. While the target rotates, the Camera rotates with it to maintain the offset and to maintain the same view of the target.
```
Comment (glabute) [inline Documentation~/CinemachineBindingModes.md:8]: _when_ the target rotates

File: Documentation~/CinemachineBindingModes.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+
+
+</br></br>
+
+### Lock To Target On Assign
+
+
+
+Makes the orientation of the virtual Camera match the local frame of the Follow target, at the moment when the virtual Camera is activated or when the target is assigned. This offset remains constant in world space. Also, the Camera does not rotate along with the target.
```
Comment (glabute) [inline Documentation~/CinemachineBindingModes.md:35]: delete _also_

File: Documentation~/CinemachineBindingModes.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+
+</br></br>
+
+
+### World Space
+
+
+
+The offset interprets the world space relative to the origin of the Follow target. The Camera will not change position when the target rotates.
```
Comment (glabute) [inline Documentation~/CinemachineBindingModes.md:67]: "The offset interprets the world space relative to the origin of the Follow target" makes no sense.  What are you trying to say here?

**Review by glabute — CHANGES_REQUESTED** [review]
I like the pictures and the concept of this page.  It's very helpful.
General remark: why is _Camera_ capitalized everywhere?  It's a bit off-putting.  It should just be _camera_.

File: Documentation~/CinemachineBindingModes.md (hunk 1 above)
Comment (bryanzakounity) [reply Documentation~/CinemachineBindingModes.md:3]: Done.

File: Documentation~/CinemachineBindingModes.md (hunk 2 above)
Comment (bryanzakounity) [reply Documentation~/CinemachineBindingModes.md:8]: Done.

File: Documentation~/CinemachineBindingModes.md (hunk 3 above)
Comment (bryanzakounity) [reply Documentation~/CinemachineBindingModes.md:35]: `Done. As well, all cameras are now lower case "c".`

File: Documentation~/CinemachineBindingModes.md (hunk 4 above)
Comment (bryanzakounity) [reply Documentation~/CinemachineBindingModes.md:67]: Yeah, that was a weird turn of phrase. Replaced with: "The offset in interpreted in world space relative to the origin of the Follow target. "

File: Documentation~/CinemachineBodyTransposer.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ The fixed offset can be interpreted in various ways, depending on the Binding Mo
 
 | **Property:** || **Function:** |
 |:---|:---|:---|
-| __Binding Mode__ || The coordinate space to use when interpreting the offset from the target, and the damping. For example, for a "Door Cam or “Hood Cam" in a car racing game, use Local Space Locked To Target. |
+| __[Binding Mode](CinemachineBindingModes.md)__ || The coordinate space to use when interpreting the offset from the target, and the damping. For example, for a "Door Cam or “Hood Cam" in a car racing game, use Local Space Locked To Target. |
```
Comment (glabute) [inline Documentation~/CinemachineBodyTransposer.md:11]: There is also a Binding Mode property in Orbital Transposer - need a similar change there

**Review by glabute — CHANGES_REQUESTED** [review]

File: Documentation~/CinemachineBodyTransposer.md (hunk 5 above)
Comment (glabute) [reply Documentation~/CinemachineBodyTransposer.md:11]: FreeLook also

Comment (bryanzakounity) [reply Documentation~/CinemachineBodyTransposer.md:11]: Linked added to those topics.

**Review by glabute — APPROVED** [review]

**Pull request merged**